fargate-eventbridge-serverless/README.md

```diff
@@ -0,0 +1,82 @@
+# Leveraging Fargate for Scheduled Jobs
+
+This pattern contains a sample cloudformation template to deploy a scheduled AWS Fargate task using EventBridge running on an Amazon Elastic Container Service (ECS) cluster. The docker image is pushed to Amazon Elastic Container Registry (ECR) using cloudformation template without having to pre-push the image to ECR or another container library. The serverless framework plugin for fargate, `serverless-fargate` is used to deploy tasks on ECS Cluster. This task runs every 10th minute of the hour and a simple file containing JSON is put in S3 bucket. 
```

Comment:
CloudFormation. 
I think it would be worth to mention that this is using serverless framework early on, as you are putting this as a CloudFormation template, and it is inside a SF one.

Reply:
Are you using EventBridge Scheduler for this?

Reply:
Thanks for the review, yes I am using Eventbridge to schedule the Fargate task